tox.ini

```diff
@@ -19,12 +20,14 @@ deps =
        -rrequirements/requirements-optionals.txt
 
 [testenv:py27-flake8]
+downloadcache={env:PIP_DOWNLOAD_CACHE}
 deps =
        -rrequirements/requirements-testing.txt
        -rrequirements/requirements-codestyle.txt
 commands = ./runtests.py --lintonly
 
 [testenv:py27-docs]
+downloadcache={env:PIP_DOWNLOAD_CACHE}
```

Comment:
Shoudn't `py27-docs` env already inherit `testenv`'s `downloadcache` attribute because of the `testenv:` prefix?